README.md

```diff
@@ -989,6 +989,92 @@ when the content is present.
   end
 ```
 
+## Custom Components
+
+When you use custom wrappers, you might also be looking for a way to add custom components to your 
+wrapper. The default components are:
+
+```ruby
+:label         # The <label> tag alone
+:input         # The <input> tag alone
+:label_input   # The <label> and the <input> tags
+:hint          # The hint for the input
+:error         # The error for the input
+```
+
+A custom component might be interesting for you if your views look something like this:
+
+```erb
+<%= simple_form_for @blog do |f| %>
+  <div class="row">
+    <div class="span1 number">
+      1
+    </div>
+    <div class="span8">
+      <%= f.input :title %>
+    </div>
+  </div>
+  <div class="row">
+    <div class="span1 number">
+      2
+    </div>
+    <div class="span8">
+      <%= f.input :body, :as => :text %>
+    </div>
+  </div>
+<% end %>
+```
+
+A cleaner method to create your views would be:
+
+```erb
+<%= simple_form_for @blog, :wrapper => :with_numbers do |f| %>
```

Comment:
Wdyt of using the new hash syntax here (since the README.md is following it):
`:wrapper => :with_numbers` -> `wrapper: :with_numbers`

Reply:
I forgot this line 😭